### PR TITLE
Pr/kamblepranay1010/101 email send

### PR DESCRIPTION
**Pull request by kamblepranay1010**
Email Sending after submit
logo

### CODE REVIEW

**Comment by nuclearsagar** [comment]
Go home

**Comment by kamblepranay1010** [comment]
9 hours not completed...

File: github.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+vdbdhdghfg
```
Comment (nuclearsagar) [inline github.txt:2]: add one more line same like this

**Review by nuclearsagar — CHANGES_REQUESTED** [review]
Do the changes again

**Review by nuclearsagar — APPROVED** [review]

**Review by pruthvipatel11 — APPROVED** [review]

**Pull request merged**